### ISSUE TITLE
Coursera | deeper understanding on lab work and practice lab

### ISSUE THREAD

**Issue by Kai0421** [Kai0421/TensorFlowLearning#10]
[Linear Regression squared error and gradient descent](https://github.com/Kai0421/PyTorchLearning/blob/main/LinearRegressionLab.ipynb)
